samples/Prisma/datasource-mongo.prisma

```diff
@@ -0,0 +1,41 @@
+datasource mongo {
```

Comment:
We use these sample files to train a Bayesian classifier. I'm worried that the 5 files you've added are more similar than usual files would be (all from similar database context). The classifier could for instance learn that `createdAt` is a good indication of a Prisma file, when that is likely not the case. Any chance you could change them to more representative sample files?

Reply:
Hey @pchaigno, sure I can definitely differentiate the schema examples!

Reply:
Thanks! The samples look better now :+1: